packages/fluentui/react-icons-northstar/src/components/NumberListIcon.tsx

```diff
@@ -6,70 +6,41 @@ import createSvgIcon from '../utils/createSvgIcon';
 const NumberListIcon = createSvgIcon({
```

Comment:
This one is special as it, instead of rendering the icon from theme, it contains the svg directly.

Reply:
Do this icon exist in `theme`? If no, this will not work with shorthand compat, icons should always be in theme for 0.47.x

Reply:
Converted to make it the same like all other icons